internal/plugin/connectors/http/generic/oauthprotocol/protocol_test.go

```diff
@@ -0,0 +1,394 @@
+package oauth1protocol
+
+import (
+	"io"
+	"io/ioutil"
+	gohttp "net/http"
+	"net/url"
+	"strings"
+	"testing"
+)
+
+func Test_extractOAuthParams(t *testing.T) {
+	type args struct {
+		params map[string]string
+	}
+	tests := []struct {
+		name string
+		args args
+		want OAuth1
+	}{
+		{
+			name: "Validate OAuth Params",
+			args: args{
+				params: map[string]string{
+					oauth_consumer_key:     "oauth_consumer_key",
+					oauth_consumer_secret:  "oauth_consumer_secret",
+					oauth_token:            "oauth_token",
+					oauth_token_secret:     "oauth_token_secret",
+					oauth_nonce:            "oauth_nonce",
+					oauth_signature:        "oauth_signature",
+					oauth_signature_method: "oauth_signature_method",
+					oauth_version:          "oauth_version",
+					oauth_timestamp:        "oauth_timestamp",
+				},
+			},
+			want: OAuth1{
+				ConsumerKey:     "oauth_consumer_key",
+				ConsumerSecret:  "oauth_consumer_secret",
+				Signature:       "oauth_signature",
+				SignatureMethod: "HMAC-SHA1",
+				TimeStamp:       "",
+				Token:           "oauth_token",
+				TokenSecret:     "oauth_token_secret",
+				Version:         "1.0",
+			},
+		},
+	}
+	for _, tt := range tests {
```

Comment:
You are running a single test using a matrix test setup. I think this can just be a single inline test.